backend/server.js

```diff
@@ -119,6 +119,15 @@ sbp('okTurtles.data/set', PUBSUB_INSTANCE, createServer(hapi.listener, {
 }))
 
 ;(async function () {
+  await initDB()
+  await sbp('chelonia/configure', {
+    acceptAllMessages: true,
+    skipActionProcessing: true,
+    skipSideEffects: true,
+    reingestEvents: false,
+    strictProcessing: true,
+    strictOrdering: true
+  })
```

Comment:
Consider adding a variable called `server: true` that inside of `'chelonia/configure'` sets the other variables accordingly, and could also be checked inside of chelonia where something must be done always on the server.